graphql/data/schema.js

```diff
@@ -738,9 +740,28 @@ const userType = new GraphQLObjectType({
       type: new GraphQLNonNull(GraphQLBoolean),
       description: 'whether a v4 user has been introduced to squads in the ui',
     },
+    userFeatures: {
```

Comment:
Naming feedback for the future: since this is a property on the user, you could just call this "features". The user connection is implied, I think.